services/credential-issuance-server/src/agent.ts

```diff
@@ -40,17 +46,51 @@ class Agent {
 
   async issueAcdcCredentialByAid(schemaSaid, aid, attribute) {
     return this.signifyApi.issueCredential(
-      Agent.ISSUER_AID_NAME,
+      Agent.HOLDER_AID_NAME,
       this.keriRegistryRegk,
       schemaSaid,
       aid,
       attribute
     );
   }
+  async issueLeChainedCredential(legalEntityAidPrefix: string) {
+    const AIDIssuerName = Agent.ISSUER_AID_NAME;
+    const AIDHolderName = Agent.HOLDER_AID_NAME;
+    const issuerAid = await this.signifyApiIssuer.getIdentifierByName(AIDIssuerName);
+    const holderAid = await this.signifyApi.getIdentifierByName(AIDHolderName);
+    const issuerAidOobi = await this.signifyApiIssuer.getOobi(issuerAid.name);
+    const holderAidOobi = await this.signifyApi.getOobi(holderAid.name);
+    await this.signifyApi.resolveOobi(issuerAidOobi);
+    await this.signifyApiIssuer.resolveOobi(holderAidOobi);
+    // Qvi
+    const qviCredentialId = await this.signifyApiIssuer.issueQVICredential(
+      issuerAid.name,
+      this.keriIssuerRegistryRegk,
+      "EBfdlu8R27Fbx-ehrqwImnK-8Cm79sqbAQ4MmvEAYqao",
+      holderAid.prefix
+    );
+    await new Promise((rs) => {
```

Comment:
Why are we waiting 5 seconds here? We should be waiting until an operation is done right?

Reply:
Yes. We should be waiting an an operation is done. Sleep 5s only for test